src/codeu/chat/server/Model.java

```diff
@@ -71,6 +72,28 @@ public int compare(Time a, Time b) {
   public void add(User user) {
     currentUserGeneration = userGenerations.make();
 
+    /// SQLite
+    Connection c = null; /// Create connection
+    Statement stmt = null;
+    try {
+      Class.forName("org.sqlite.JDBC");
+      c = DriverManager.getConnection("jdbc:sqlite:test.db"); /// Connect to database
+      c.setAutoCommit(false); 
+      System.out.println("Opened database successfully model");
+
+      stmt = c.createStatement();
+      ///Test data to be entered into table
+      String sql = "INSERT INTO USER (ID,TIME,NAME) " +  
```

Comment:
This database name should be defined as a constant, so that you can be sure that the same name is used across files.